src/web/components/HeadlineByline.tsx

```diff
@@ -126,14 +133,15 @@ export const HeadlineByline = ({ format, byline, tags }: Props) => {
 				case Design.Comment:
 					return (
 						<div
-							className={`${opinionStyles(palette, format)} ${
-								tags.filter((tag) => tag.type === 'Contributor')
-									.length === 1
-									? authorBylineWithImage
-									: ''
-							}`}
+							className={cx(opinionWrapperStyles, {
+								[authorBylineWithImage]: hasSingleContributor(
+									tags,
+								),
+							})}
 						>
-							<BylineLink byline={byline} tags={tags} />
+							<div className={opinionStyles(palette, format)}>
```

Comment:
Is there a way to merge this with the outer div? I suspect not from reading your description but if there were it's always nice to have fewer divs!

Great work by the way - thanks for fixing my glitch! Ideally one day we could use something really nice like `shape-outside` I think I did look at this way back and never fully discounted it, I just didn't have the weight of experience to know how useful it would have been and that it was worth the extra time to implement.

Reply:
I think there might be if we could add the special report highlighting to the link itself rather than the wrapper - but the byline is not always just a link, it could have other text like `and` in the middle for multiple contributors.

Reply:
Maybe a span in the `BylineLink` component, but I think you have an extra level of nesting somewhere no matter what?